README.md

```diff
@@ -1,24 +1,138 @@
 
-* Configuration
+***The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.***
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- ORM model
+- User, Food, Recipe Food, and Recipe model
 
-* ...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal
+- IDE such as VsCode.
+- Rails needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example command:
+
+```sh
+  git clone https://github.com/chaseknis/recipe_app.git
+  cd recipe_app
+
+```
+
+### Install
+
+Example command:
+
+```sh
+  cd recipe_app
+  bundle install
+
+```
+
+### Usage
+
+To run the project redirecting to the root directory of the project and run the following command:
+
+```sh
+ bin/rails server 
+
+```
+
+### Test
+
+To test the web app run the following command:
+
+```sh
+ rspec spec
+
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Innocent N.**
+
+- GitHub: [@chaseknis](https://github.com/Chaseknis)
+- Twitter: [@ChaseToTheWorld](https://twitter.com/chasetotheworld)
+- LinkedIn: [Innocent N.](https://www.linkedin.com/in/innocent-n-200826252/)
+
+👤 Raphael Okolo
+
+- GitHub: [@RaphOkolo](https://github.com/RaphDasilva)
+- Twitter: [@RaphOkolo](https://twitter.com/RaphOkolo)
+- LinkedIn: [@RaphOkolo](https://www.linkedin.com/in/raphael-okolo/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Validations and Model specs.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chaseknis/recipe_app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse community for the support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to add a LICENSE to your repo, as for now this link redirect to a 404 page